### ISSUE TITLE
Add full support for emojis

### ISSUE THREAD

**Issue by sdw4938** [windows-server-2003/ThirdTube#35]
Emojis would be nice. Without the emojis there are question marks in their place.

**Comment by Poketubepoggu** [comment]
That's odd. On my end emojis work fine.

**Comment by NetherMCtv** [comment]
yes emojis are working but some of these are not showing, these showing a broken character
